en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-docker.md

```diff
@@ -59,90 +55,87 @@ The project is now initialized. A directory with the name "petstore" has been cr
+Start Choreo Connect on Docker by executing the Docker Compose script inside the extracted distribution folder by navigating to `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/` and executing the following command.
 
 {!includes/deploy/cc-tryout-in-arm64-docker-note.md!}
 
 ``` java
 docker-compose up -d
 ```
 
-Once containers are up and running, we can monitor the status of the containers using the following command
+Once the containers are up and running, we can monitor the status of the containers using the following command.
     ``` java
     docker ps | grep choreo-connect-
     ```
 
 ## Step 4 - Deploy the API Project
 
-### Step 4.1 - Add Choreo Connect Cluster as Environment to APICTL
+1. Add the Choreo Connect Cluster as an Environment to apictl
 
-To use APICTL with Choreo Connect, we need to add the Choreo Connect cluster as an environment in the APICTL.
-Basically the adapter URL will be added as the gateway environment, and the added environment can be used in the subsequent commands.
+    To use apictl with Choreo Connect, we need to add the Choreo Connect cluster as an environment to apictl.
+    Basically the adapter URL will be added as the gateway environment, and the added environment can be used in the subsequent commands.
 
-``` shell tab="Format"
-apictl mg add env <ENVIRONMENT_NAME> --adapter <ADAPTER_URL>
-```
+    ``` shell tab="Format"
+    apictl mg add env <ENVIRONMENT_NAME> --adapter <ADAPTER_URL>
+    ```
 
-``` shell tab="Example"
-apictl mg add env dev --adapter https://localhost:9843
-```
+    ``` shell tab="Example"
+    apictl mg add env dev --adapter https://localhost:9843
+    ```
 
-### Step 4.2 - Login to Choreo Connect Cluster
+2. Log in to the Choreo Connect Cluster
 
-Next you need to log in to the Choreo Connect environment (log in to the adapter) in order to deploy the API in Choreo Connect.
+    Next you need to log in to the Choreo Connect environment (log in to the adapter) in order to deploy the API in Choreo Connect.
 
-``` shell tab="Format"
-apictl mg login dev -u <AUTHORIZED_USER_USERNAME> -p <USER_PASSWORD> -k
-```
+    ``` shell tab="Format"
+    apictl mg login dev -u <AUTHORIZED_USER_USERNAME> -p <USER_PASSWORD> -k
+    ```
 
-``` shell tab="Example"
-apictl mg login dev -u admin -p admin -k
-```
+    ``` shell tab="Example"
+    apictl mg login dev -u admin -p admin -k
+    ```
 
-!!! info
-    Following apictl commands are being executed with -k flag to avoid SSL verification with the Choreo Connect.
-    To communicate via https without skipping SSL verification (without -k flag), add the cert of Choreo Connect into `/home/<your-pc-username>/.wso2apictl/certs`.
+    !!! info
+        The following apictl commands are executed with the flag `-k` to avoid the SSL verification with Choreo Connect. To communicate via https without skipping the SSL verification (without -k flag), add the cert of Choreo Connect into `/home/<your-pc-username>/.wso2apictl/certs`.
```

Comment:
```suggestion
        The following apictl commands are executed with the flag `-k` to avoid the SSL verification with Choreo Connect. To communicate via HTTPS without skipping the SSL verification (without -k flag), add the cert of Choreo Connect into `/home/<your-pc-username>/.wso2apictl/certs`.
```